### ISSUE TITLE
Cannot re-arrange items when Mac VoiceOver is running

### ISSUE THREAD

**Issue by jamiek-acl** [atlassian/react-beautiful-dnd#1622]
This is related to https://github.com/atlassian/react-beautiful-dnd/issues/1273, but that issue is for iOS and I am talking about desktop OS.

### Expected behavior
That it is possible to reorder/drag items with voiceover turned on.

### Actual behavior
It tells me when I picked up an item, it tells me when I dropped an item, but the items do not actually re-order.

### Steps to reproduce
Go to the demo: https://react-beautiful-dnd.netlify.com/iframe.html?id=board--simple
Turn on VoiceOver (cmd + f5)
Tab to the first item in a list of multiple items
Press spacebar (the screenreader says "You have lifted an item in position 1")
Press down arrow a few times
Press spacebar (the screenreader says "You have dropped the item. You have moved it from position 1 to position 1")

### What browser are you using?
Tried in Chrome 78.0.3904.97, Safari 13.03

**Comment by mendeza** [comment]
I'm dealing with this too. I suspect that an experienced VO user probably knows how to configure the utility to effect dropping, but I haven't sussed it out yet. Having similar issues on Windows with NVDA and JAWs. Did you ever find a resolution, @jamiek-acl?

**Comment by jamiek-acl** [comment]
@mendeza oh man this was 3 years ago, so i cant remember.  However it seems to be working in our component with VoiceOver turned on (it is using react-beautiful-dnd v13.0.0):
https://paprika.highbond.com/?path=/story/forms-sortable-examples--functional

**Comment by mendeza** [comment]
Hey, thanks for the speedy reply, @jamiek-acl. Nice work on Paprika! However on my system I still experience the problem on the page you shared. This makes me suspect even more that it's a local screen reader configuration issue that I just haven't sussed-out yet.

**Comment by jamiek-acl** [comment]
@mendeza Really it doesnt work for you?
I dont think I've ever changed any screen reading configuration.
Using latest Chrome (v106.0.5249.91)
Make sure you are on the "Functional" story, the other ones dont update state: https://paprika.highbond.com/?path=/story/forms-sortable-examples--functional
Turn on screen reader
Click on the frame where the story is to save some tabs
Tab to first item (Lorem hipsum)
Press space, it says "You have listed the item in position 1"
Press down arrow twice, it says "You have moved the item from position 1 to position 3"
Press space, it says "You have dropped the item. It has moved from position 1 to 3"
